src/views/Brokering.vue

```diff
@@ -150,67 +152,28 @@ export default defineComponent({
   },
   methods: {
     async addBatch() {
-      const batchmodal = await modalController.create({
-        component: BatchModal
-      });
-      return batchmodal.present();
-    },
-    async editBatch(id: string, enumId: string) {
       const batchmodal = await modalController.create({
         component: BatchModal,
-        componentProps: {id, enumId}
+        breakpoints: [0, 0.25, 0.5, 0.75, 1],
+        initialBreakpoint: 1
       });
       return batchmodal.present();
     },
-    async deleteBatch(batch: any) {
-      const deleteBatchAlert = await alertController
-        .create({
-          header: this.$t('Cancel job'),
-          message: this.$t("Canceling this job will cancel this occurrence and all following occurrences. This job will have to be re-enabled manually to run it again."),
-          buttons: [
-            {
-              text: this.$t("Don't cancel"),
-              role: 'cancel',
-            },
-            {
-              text: this.$t("Cancel"),
-              handler: async () => {
-                await this.store.dispatch('job/cancelJob', batch);
-              },
-            },
-          ],
-        });
-      return deleteBatchAlert.present();
-    },
-    async skipBatch (batch: any) {
-      const skipJobAlert = await alertController
-        .create({
-          header: this.$t('Skip job'),
-          message: this.$t('Skipping will run this job at the next occurrence based on the temporal expression.'),
-          buttons: [
-            {
-              text: this.$t("Don't skip"),
-              role: 'cancel',
-            },
-            {
-              text: this.$t('Skip'),
-              handler: async () => {
-                this.store.dispatch('job/skipJob', batch).then((resp) => {
-                  if (resp.status === 200 && !hasError(resp)) {
-                    showToast(translate("This job has been skipped"));
-                  } else {
-                    showToast(translate("This job schedule cannot be skipped"));
-                  }
-                });
-                (this as any).$refs.slidingOptions.$el.closeSlidingItems();
-              },
-            }
-          ]
-        });
-      return skipJobAlert.present();
-    },
     getTime (time: any) {
-      return DateTime.fromMillis(time).toLocaleString(DateTime.DATETIME_MED);
+      return DateTime.fromMillis(time).toFormat('hh:mm a');
+    },
+    getBrokerQueue(job: any){
+      if(job.status === 'SERVICE_PENDING'){
```

Comment:
As we have already added check for SERVICE_PENDING in the template we don't need to check the same condition again.